pkg/annotations/ingress_annotations.go

```diff
@@ -0,0 +1,51 @@
+package annotations
+
+import (
+	extensions "k8s.io/api/extensions/v1beta1"
+)
+
+const (
+	// Prefix AppGw Prefix
+	Prefix = "appgw.ingress.kubernetes.io"
+
+	// BackendPathPrefix defines the value for Path which should be used as a prefix for all HTTP requests.
+	// Null means no path will be prefixed. Default value is null.
+	BackendPathPrefix = "appgw.ingress.kubernetes.io/backend-path-prefix"
```

Comment:
Also please use the "Prefix" in subsequent definition such as in BackendPathPrefix .